checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInference.java

```diff
@@ -441,10 +558,10 @@ private void addEnsuresCalledMethods() {
   /**
    * Possibly adds an InheritableMustCall annotation on the enclosing class.
    *
-   * <p>If the class already has a non-empty MustCall type (that is inherited from one of its
-   * superclasses), this method does nothing, in order to avoid infinite iteration. Otherwise, if
-   * the current method is not private and satisfies the must-call obligations of all the owning
-   * fields, it adds (or updates) an InheritableMustCall annotation to the enclosing class.
+   * <p>Let the enclosing class be C. If C already has a non-empty MustCall type (that is inherited
+   * from one of its superclasses), this method does nothing, in order to avoid infinite iteration.
+   * Otherwise, if the current method is not private and satisfies the must-call obligations of all
+   * the owning fields in C, it adds (or updates) an InheritableMustCall annotation to C.
    */
   private void addOrUpdateClassMustCall() {
```

Comment:
Why is the name "addOrUpdate"?  The first sentence of the documentation lacks "update", and I don't see that it adds much (if anything) to the clarity at call sites.